docs/navbar.html

```diff
@@ -0,0 +1,20 @@
+<div class="navbar">
+  <h1 id="nav-header" class="nav-header"></h1>
+
+  <nav role="navigation">
+    <div id="nav-toggle">
+      <!-- A hidden checkbox just to be the click receiver the :checked selector can be used. -->
+      <input id="nav-input" type="checkbox" />
+
+      <!-- Hamburger. -->
+      <span></span>
+      <span></span>
+      <span></span>
+
+      <ul class="nav-menu">
+        <a href="/"><li>Map</li></a>
+        <a href="/import/add_asset.html"><li>Upload Asset</li></a>
```

Comment:
Should probably link to add disaster page? I think this upload page might actually be less useful with some recent changes. Isn't it actually broken?

Reply:
Switched